core/adapters/bigquery.ts

```diff
@@ -89,4 +90,24 @@ export class BigQueryAdapter extends Adapter implements IAdapter {
   public dropIfExists(target: dataform.ITarget, type: string) {
     return `drop ${this.baseTableType(type)} if exists ${this.resolveTarget(target)}`;
   }
+
+  public insertInto(
```

Comment:
R.e. my comment in the design doc, can we stick with insert when no uniqueKey is specified?